### DISCUSSION TITLE
Disable Update Notifications for Recipient Accounts

### DISCUSSION THREAD

**Discussion by Philc-215** [globaleaks/globaleaks-whistleblowing-software#4138] [Ideas]
This may be possible already and I have missed something, but I will post anyway!

We have a Primary Recipient account that receives all email and update notifications to a Shared Mailbox. we then have a number of standard Recipient accounts that are getting the updates twice (once in the central mailbox and once in their personal mailbox)

We have disabled Email Notifications on these accounts but there is no option to disable Notifications for specific accounts, just the entire Recipient Role which would prevent the Primary account from getting the updates.

Is there a solution to this issue or could it be a future option, 

and does this make any sense!

**Comment by evilaliv3** [comment]
@Philc-215 : on any user you could disable notifications selectively.

Try to configure a user and you will find the option.
